### PR TITLE
Updated enroll endpoints and added verify endpoints

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Updated enrollment endpoints and added verification endpoints
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-428184](https://oktainc.atlassian.net/browse/OKTA-428184)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 1]
```diff
@@ -96,16 +96,23 @@ API endpoints that take username and password credentials, including the [Authen
 
 > **Note:** Okta round-robins between SMS providers with every resend request to help ensure delivery of SMS OTP across different carriers.
 
-* **Enrollment rate limit:** This rate limit applies to a user's attempt to enroll an [SMS or a Call factor](/docs/reference/api/factors/) using any phone number. This rate limit applies to only the enrollment operation. See [System Log events for rate limits](/docs/reference/rl-system-log-events/#debugcontext-object-examples) for examples of System Log rate limit events where too many enrollment attempts for the SMS or Call factors were made.
+* **Enrollment and verification rate limits:** This rate limit applies to a user's attempt to enroll or verify an [SMS or a Call factor](/docs/reference/api/factors/) using any phone number. This rate limit applies only to the enrollment or verification operation. See [System Log events for rate limits](/docs/reference/rl-system-log-events/#debugcontext-object-examples) for examples of System Log rate limit events where too many enrollment or verification attempts for the SMS or Call factors were made.
 
-  **Endpoints**
+  **Enroll Endpoints**
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:101]:
```suggestion
  **Enroll endpoints**
```

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 2]
```diff
@@ -96,16 +96,23 @@ API endpoints that take username and password credentials, including the [Authen
...
-  **Identity Engine endpoints**<br>
-  <ApiLifecycle access="ie" />
-  * `/idp/idx/challenge`
-  * `/idp/idx/credential/enroll`
+  * `/api/v1/authn/factors/${id}/lifecycle/resend`
+  * `/api/v1/users/${userId}/factors/${userFactorId}/lifecycle/activate/sms`
+  * `/api/v1/users/${userId}/factors/${userFactorId}/resend`
+  * `api/${apiVersion}/radius`
+
+  **Verify Endpoints**
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:109]:
```suggestion
  **Verify endpoints**
```

**Review by vanngo-okta — APPROVED** [review]
Small nits. Otherwise, LGTM!

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 3]
```diff
@@ -96,16 +96,23 @@ API endpoints that take username and password credentials, including the [Authen
...
+  * `/api/v1/users/${userId}/factors/${userFactorId}/resend`
+  * `api/${apiVersion}/radius`
+
+  **Verify endpoints**
+  * `/api/v1/authn/factors/${factorIdOrFactorType}/verify`
+  * `/api/v1/authn/factors/${factorIdOrFactorType}/verify/resend`
+  * `/api/v1/users/${userId}/factors/${userFactorIdOrFactorType}/verify`
+  * `/auth/call/send`
+  * `/auth/sms/send`
+  * `api/${apiVersion}/radius`
```
Comment (scottbermon-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:115]: @christinechoy-okta, after discussions with our Product lead (Rebecca Boureston), we'd prefer to not list the individual endpoints that we are rate limiting for SMS and VOICE.

If we could remove these lines that list the endpoints and retain the above (line 99) paragraph noting the Enrollment and Verification rate limiting occurs on SMS/VOICE, that should be sufficient for this Pull Request to be approved.

**Pull request closed without merging**